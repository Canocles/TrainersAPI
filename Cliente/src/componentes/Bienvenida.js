import React from 'react'

class Bienvenida extends React.Component {
    render() {
        return (
            <div className="container" style={{ padding: 10 }}>
                <div className="jumbotron">
                    <h1>Bienvenido a la PÃ¡gina Principal de Trainners API</h1>
                </div> 
            </div>
        )    
    }
}

//Exportamos el componente. Usamos el "default"
//porque no necesitamos exportar varias cosas separadas
export default Bienvenida