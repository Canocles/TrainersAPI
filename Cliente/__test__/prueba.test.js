import React from 'react';
import renderer from 'react-test-renderer'
import Bienvenida from '../src/componentes/Bienvenida';

describe('Bienvenida (snpashot)', () => {
  it('La pÃ¡gina de Bienvenida se renderiza correctamente', () => {
    const component = renderer.create(<Bienvenida />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
