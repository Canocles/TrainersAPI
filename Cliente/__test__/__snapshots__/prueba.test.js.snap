// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bienvenida (snpashot) La página de Bienvenida se renderiza correctamente 1`] = `
<div
  className="container"
  style={
    Object {
      "padding": 10,
    }
  }
>
  <div
    className="jumbotron"
  >
    <h1>
      Bienvenido a la Página Principal de Trainners API
    </h1>
  </div>
</div>
`;
